_tutorials/2016-05-10-iosxr-ansible-md.md

```diff
@@ -1,32 +1,32 @@
 ---
-permalink: "/tutorials/IOSXR-Ansible"
+permalink: "/tutorials/XR-Ansible"
 author: Mike Korshunov
-excerpt: Getting started with IOSXR and Ansible playbooks
+excerpt: Getting started with XR and Ansible playbooks
 published: true
-title: "IOSXR-Ansible.md"
+title: "IOSXR-Ansible"
 ---
 # Introduction
-In this tutorial we are going to cover XRv4 configuration with Ansible. We will setup and try simplest configuration with Unix machine connected to XRv64.
+In this tutorial we are going to cover IOS-XRv configuration with Ansible. We will setup and try simplest configuration with Unix machine connected to IOS-XRv.
```

Comment:
We will set up and try a very simple configuration with a linux machine connected to IOS-XRv.